README.md

```diff
@@ -218,7 +218,9 @@ http {
 
         -- errs is an array-like table
         if errs then
-            for i = 1, #errs do ngx.log(ngx.ERR, errs[i])
+            for i = 1, #errs do
+	        ngx.log(ngx.ERR, errs[i])
+	    end
```

Comment:
to be pedantic, mind if we align this `end` with the `for` above?

Reply:
I will double-check but I thought it was already aligned.

Reply:
Yeah, in the whole-file view mode it looks aligned: https://github.com/kwaping/lua-resty-waf/blob/77006f326aa4bb9406091a2d74435c7faa0d26a8/README.md

Reply:
Not particularly interested in whole-file view :) We can clearly see these are misaligned. tabs vs spaces perhaps?